code-challenges.test.js

```diff
@@ -22,24 +30,96 @@ const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
 // b) Create the function that makes the test pass.
+// Input: takes in a string
+// Output: retunrs a string that has been encrypted.
+// Psuedo Code: create a function that turn my string into a array so i can change aech value per index using .split to break it into an array. use an if statement to iterate thru the array and change the value based on {  "a" to 4, "e" to 3, "i" to 1, and "o" to 0.  } 
+
+
+
+const theCoder = (string)=> {
+    newArray = string.split('')
+    
+    
+    const modifiedArray = newArray.map((char) => {
+        if (char == "a") {
+          return "4";
+        } else if (char == "E") {
+          return "3";
+        } else if (char == "i") {
+          return "1";
+        } else if (char == "o") {
+          return "0";
+        } else if (char == "e") {
+            return "3";
+        } else {
+          return char;
+        }
+      });
```

Comment:
I love this approach with a conditional statement. Why did you choose to use loose equality?

Reply:
Its honestly confusing to remember ruby and javascript ===. I'm pretty sure over time, with  more practice it will get there